webui/react/src/e2e/fixtures/global-fixtures.ts

```diff
@@ -69,18 +82,28 @@ export const test = baseTest.extend<CustomFixtures, CustomWorkerFixtures>({
     await use(apiAuth);
   },
 
-  apiProject: async ({ apiAuth }, use) => {
-    const apiProject = new ApiProjectFixture(apiAuth);
+  apiProject: async ({ apiArgs }, use) => {
+    const apiProject = new ApiProjectFixture(apiArgs);
     await use(apiProject);
   },
 
-  apiUser: async ({ apiAuth }, use) => {
-    const apiUser = new ApiUserFixture(apiAuth);
+  apiSearches: async ({ apiWorkspace, apiProject, apiArgs }, use) => {
+    // TODO: save everyone some time by having new call the create api
+    const workspaceProps = apiWorkspace.new();
+    const workspace = await apiWorkspace.createWorkspace(workspaceProps);
+    const projectProps = apiProject.new({ projectProps: { workspaceId: workspace.workspace.id } });
+    const project = await apiProject.createProject(projectProps.workspaceId, projectProps);
+    await use(new ApiSearchFixture(apiArgs, project.project.id));
+    await apiWorkspace.deleteWorkspace(workspace.workspace.id);
```

Comment:
q: why are we creating a new workspace and then pretty much immediately deleting it? when do the tests get run?